tests/integration-tests/tests/budgets/test_budgets.py

```diff
@@ -33,6 +37,50 @@ def test_update_budget_properties(pcluster_config_reader, clusters_factory, test
     # Add one budget of each BudgetCategory, update them, and check that the fields match the expected values
     _test_update_cluster_and_check_fields(cluster, budget_client, account_id, region, pcluster_config_reader)
 
+    # Enable fleet stop action, trigger sns topic notification, and assert that the fleet has stopped
+    _test_stop_compute_fleet(cluster, pcluster_config_reader)
+
+
+def _test_stop_compute_fleet(cluster, pcluster_config_reader):
+    # Create the initial cluster with a budget triggered compute-fleet stop action
+    stop_fleet_config_file = pcluster_config_reader(config_file="budget_with_fleet_stop.yaml")
+    cluster.update(str(stop_fleet_config_file), force_update="true")
+    time.sleep(30)
```

Comment:
Instead of adding time sleep here. Lets use the retry and wait like what you do in line 79-83.

Reply:
As we discussed, it was removed. The compute fleet would already be running after cluster update.